src/components/Modal/Modal.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Styled from 'styled-components';
+
+export default function Modal({ on, setOn, children }) {
+  return (
+    <ModalStyle style={!on ? { display: 'none' } : { display: 'block' }}>
```

Comment:
styled-component를 사용하시면 더 좋은 방법이 생깁니다!!
```suggestion
    <ModalStyle display={on}>
```
이런식으로 props를 넘겨서 사용해볼까요?